packages/ozone/src/api/proxied.ts

```diff
@@ -127,4 +127,18 @@ export default function (server: Server, ctx: AppContext) {
       }
     },
   })
+
+  server.app.bsky.graph.getLists({
+    auth: ctx.authVerifier.moderator,
```

Comment:
No real issue here, but if there's no special functionality or access being provided by Ozone, it's always possible for the UI to go directly to the appview.

Reply:
good point. I was thinking that we'd want this to circumvent the takedown check so that lists from takendown account AND takendown lists both are visible when queried via ozone but forgot to dig deeper into that. let me give that another spin and see if I can get the takedown bypass in too.